posix.mak

```diff
@@ -370,6 +372,13 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 	    RET=$$? ; rm -rf $$T ; exit $$RET                                                                   \
 	  )
 
+# Target for quickly debugging a single module
+# For example: DEBUGGER=ddd BUILD=debug make -f posix.mak std/format.debug
```

Comment:
How about setting a default value for `DEBUGGER`?
Imho `gdb` would be a reasonable default.